File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceSlugIndex.cs
using System;
namespace ConferenceManagement.Domain.Models
{
    public class ConferenceSlugIndex
    {
        public string IndexId { get; private set; }
        public Guid ConferenceId { get; private set; }
        public string Slug { get; private set; }

        public ConferenceSlugIndex(string indexId, Guid conferenceId, string slug)
        {
            IndexId = indexId;
            ConferenceId = conferenceId;
            Slug = slug;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/models/ConferenceSlugIndex.java
package com.microsoft.conference.management.domain.models;

public class ConferenceSlugIndex {
    public String indexId;
    public String conferenceId;
    public String slug;

    public ConferenceSlugIndex(String indexId, String conferenceId, String slug) {
        this.indexId = indexId;
        this.conferenceId = conferenceId;
        this.slug = slug;
    }
}
